Extension/src/LanguageServer/client.ts

```diff
@@ -705,12 +705,11 @@ class DefaultClient implements Client {
     }
 
     private updateInactiveRegions(params: InactiveRegionParams): void {
-        let renderOptions: vscode.DecorationRenderOptions = {
-            light: { color: "rgba(175,175,175,1.0)" },
-            dark: { color: "rgba(155,155,155,1.0)" },
-            rangeBehavior: vscode.DecorationRangeBehavior.ClosedOpen
```

Comment:
do we need to keep the ClosedOpen rangeBehavior?

Reply:
Re-added!